src/reducers/index.ts

```diff
@@ -433,7 +441,7 @@ const appStateReducer = (state: AppState = defaultState, action: Actions): AppSt
 
 class FelfelePersistConfig implements PersistConfig {
     public transforms = [immutableTransformHack({
-        whitelist: ['contentFilters', 'feeds', 'ownFeeds', 'rssPosts', 'localPosts', 'postUploadQueue'],
+        whitelist: ['contentFilters', 'feeds', 'ownFeeds', 'rssPosts', 'localPosts', 'postUploadQueue', 'postCommandLog'],
```

Comment:
We shouldn't need the immutableTransformHack on postCommandLog

Reply:
fixed